retrofit2/build.gradle

```diff
@@ -0,0 +1,13 @@
+archivesBaseName = 'armeria-retrofit2'
```

Comment:
> Can you add a (at least simple) README with some usage?

How about adding a page to the sphinx site? We could add a link to that page later.